src/internal_events/common.rs

```diff
@@ -25,6 +25,11 @@ impl InternalEvent for EndpointBytesReceived<'_> {
             "protocol" => self.protocol.to_owned(),
             "endpoint" => self.endpoint.to_owned(),
         );
+        histogram!(
+            "component_received_event_size_bytes", self.byte_size as u64,
```

Comment:
Actually, and I'm not sure how I missed this, I don't think this name is correct. It should be `component_received_bytes`.

Reply:
Done. Thanks!